segmentation/Shanbhag.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
```

Comment:
Can you change the file name to shanbhag (lower case s), so that it consistent with Python's naming convention?